fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/DimensionRow.java

```diff
@@ -16,6 +17,21 @@
 
     private final String keyValue;
 
+
+    /**
+     * Build a dimension row with a key field value and a map of field values.
```

Comment:
Looks like this constructor only builds a singleton map. Should we say instead something like "Build a dimension row with a key field value and a **field value**"